GPU/TPCFastTransformation/macro/TPCFastTransformInit.C

```diff
@@ -125,12 +209,21 @@ void TPCFastTransformInit(const char* fileName = "debugVoxRes.root",
   // ntuple with the input data: voxel corrections
   debugFile->cd();
   TNtuple* debugVox =
-    new TNtuple("vox", "vox", "iRoc:iRow:x:y:z:vx:vy:vz:cx:cy:cz");
+    new TNtuple("vox", "vox", "iRoc:iRow:n:x:y:z:vx:vy:vz:cx:cy:cz");
 
   debugVox->SetMarkerStyle(8);
   debugVox->SetMarkerSize(0.8);
   debugVox->SetMarkerColor(kBlue);
 
+  // duplicate of debugVox
+  debugFile->cd();
+  TNtuple* debugCorrVox =
+    new TNtuple("corrvox", "corrvox", "iRoc:iRow:n:x:y:z:vx:vy:vz:cx:cy:cz");
+
+  debugCorrVox->SetMarkerStyle(8);
+  debugCorrVox->SetMarkerSize(0.8);
+  debugCorrVox->SetMarkerColor(kMagenta);
+
```

Comment:
Why is this added? It indeed seems to be an exact copy of `debugVox` filled with the same entries below.

Reply:
The only reason is to have the tree in two different colors and marker sizes.  One can then draw vx,vy, and vz in red color and cx,cy, and cz in magenta without much typing in the command line. One can put these values in different trees, but sometimes I want to see cx-vx, cy-vy etc.